src/Components/Web.JS/package.json

```diff
@@ -21,6 +24,7 @@
     "@typescript-eslint/parser": "^1.5.0",
     "eslint": "^5.16.0",
     "jest": "^24.1.0",
+    "rimraf": "^2.6.2",
```

Comment:
Doesn't this require a yarn.lock update?

Reply:
Seems like it was already being brought in transitively? In any case, I ran a `yarn upgrade rimraf` to be sure.